src/snapshotVersion.js

```diff
@@ -1,11 +1,19 @@
 const program = require('commander');
 const createVersion = require('./lib/createVersion.js');
+const fs = require("fs");
 
 module.exports = (argsParser = process.argv) => {
     program
-        .command('create <version> [siteDir]')
-        .action((version, siteDir = ".") => {
-            createVersion.create(version, siteDir);
+        .command('create')
+        .storeOptionsAsProperties(true)
+        .requiredOption('--version <type>', '[required] The value of the new version to create.')
+        .option('--siteDir <type>', '[optional] The absolute or relative path to the `website` directory of the Docusaurus V1 project.', '.')
+        .option('--staticDir <type...>', '[optional] The static asset directory to be versioned. Only the name of the folder is required.', [])
```

Comment:
Let's clarify the description of this parameter because it's expecting dir name relative to <siteDir>/static dir